src/transactions/FeeBumpTransactionFrame.cpp

```diff
@@ -194,6 +194,8 @@ FeeBumpTransactionFrame::commonValidPreSeqNum(AbstractLedgerTxn& ltx)
     auto v2 = bigMultiply(mInnerTx->getFeeBid(), getMinFee(lh));
     if (v1 < v2)
     {
+        getResult().feeCharged =
```

Comment:
It is not at all obvious to me that this one can't overflow, even if you use the bound of approximately 50B native asset. The reason is that the `baseFee` could conceivably be `UINT32_MAX`, which makes `50B * UINT32_MAX * 100` the approximate worst case value of `v2`.

Reply:
Theoretically possible sure. I'll clamp it the same way for clarity.